pkg/kube/pod.go

```diff
@@ -58,6 +58,8 @@ type PodOptions struct {
 	Namespace          string
 	ServiceAccountName string
 	Volumes            map[string]string
+	PodSecurity        *v1.PodSecurityContext
+	ContainerSecurity  *v1.SecurityContext
```

Comment:
To avoid confusion, should we name the fields `PodSecurityContext` and `ContainerSecurityContext`?